src/core/lib/iomgr/ev_epoll1_linux.c

```diff
@@ -130,9 +130,9 @@ static void fd_global_shutdown(void);
  * Pollset Declarations
  */
 
-typedef enum { UNKICKED, KICKED, DESIGNATED_POLLER } kick_state;
+typedef enum { UNKICKED, KICKED, DESIGNATED_POLLER } kick_state_t;
```

Comment:
Just curious, what was the motivation for going from kick_state to kick_state_t ? Is this a style guide rule?

Reply:
And likewise with the other places where `_t` is getting appended to struct or enum names.

Reply:
C++ does not allow same type and variable names. There were cases such as :
kick_state kick_state;
Adding a _t seemed like the easiest way to solve them.

Reply:
Very neat.